MAUI/Maps/zoom-pan.md

```diff
@@ -0,0 +1,633 @@
+---
+layout: post
+title: Zooming and Panning  in Maps control | Syncfusion
+description: Learn here all about the Shape Sublayer feature of the Syncfusion .NET MAUI Maps (SfMaps) control to customize their appearances and more.
+platform: MAUI
+control: SfMaps
+documentation: ug
+---
+
+# Zooming and Panning in .NET MAUI Maps (SfMaps)
+
+It is possible to zoom in and out for any layer to take a closer look at a specific region by pinching the map, scrolling the mouse wheel or track pad. Pan the map to navigate across the regions. You can also customize the zoom level of the rendering.
+
+The procedure for zooming and panning for both layers is very similar.
+
+**Shape layer**
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+     <map:SfMaps.Layer>
+         <map:MapShapeLayer ShapesSource="https://cdn.syncfusion.com/maps/map-data/world-map.json">
+              <map:MapShapeLayer.ZoomPanBehavior>
+                 <map:MapZoomPanBehavior ZoomLevel = "1" />
+              </map:MapShapeLayer.ZoomPanBehavior>
+         </map:MapShapeLayer>
+     </map:SfMaps.Layer>
+ </map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps maps = new SfMaps();
+MapShapeLayer layer = new MapShapeLayer();
+layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/world-map.json"));
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 1;
+layer.ZoomPanBehavior = zoomPanBehavior;
+maps.Layer = layer;
+this.Content = maps;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+**Tile layer**
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png">
+              <map:MapTileLayer.ZoomPanBehavior>
+                 <map:MapZoomPanBehavior ZoomLevel = "1" />
+              </map:MapTileLayer.ZoomPanBehavior>
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 1;
+tileLayer.ZoomPanBehavior = zoomPanBehavior;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Osm map default](images/zoom-pan/osm_map.png)
+
+## Customizing the zoom level
+
+You can set the zoom level of the map layer by using `MapZoomPanBehavior.ZoomLevel` property.
+
+The default `MapZoomPanBehavior.ZoomLevel` value is 1 which will show the whole map in the viewport for `MapTileLayer`.
```

Comment:
The default `MapZoomPanBehavior.ZoomLevel` value is 1, which will show the whole map in the viewport for `MapTileLayer.`

Reply:
Changed the content